src/plugins/pluginMFManifest.ts

```diff
@@ -304,6 +316,7 @@ const Manifest = (): Plugin[] => {
         },
         remoteEntry,
         ssrRemoteEntry: remoteEntry,
+        varRemoteEntry,
```

Comment:
@ScriptedAlchemy wdyt about additional `varRemoteEntry` field in `manifest.json`?